index.html

```diff
@@ -221,20 +221,20 @@ <h2 class="foot-section-h">Get started</h2>
 
 
     <form action="https://formspree.io/f/xaykaovn" method="post">
-        <div class="your-name-container" >
+		<div class="your-name-container" >
             <div>Your name</div>
-            <input type="text" name="your name" placeholder="Anastasiia" maxlength='30' required id="">
+            <input type="text" name="" placeholder="Anastasiia" maxlength='30'  id="">
         </div>
 
 
         <div class="email-container">
             <div>Your email</div>
-            <input type="email" placeholder="|"  name="your email" id="" required >
+            <input type="email" placeholder="|"  name="your email" id=""  >
```

Comment:
- Hey, Please, kindly ensure the email input field is required to enable the form validation.